samples/test/util/toQueryParams.js

```diff
@@ -0,0 +1,18 @@
+
+export default function toQueryParams(obj) {
```

Comment:
from the name, I was expecting the output to be an object. how about `toQueryString`?